ops/model.py

```diff
@@ -3152,6 +3152,33 @@ def get_notices(
             keys=keys,
         )
 
+    def get_identities(self) -> Dict[str, pebble.Identity]:
```

Comment:
Let's go with James's suggestion and not add these to Container for now. I don't think they'll be commonly-used.